Pipeline/D - Model sensitivity analyses and checks/D03 - Additional robustness.R

```diff
@@ -231,11 +208,14 @@ g
 complete = complete |> mutate(post1995 = (yearnum>=1995))
 complete %>% count(post1995)
 
-# formula (remove intervention dummies for temporal subsampling)
-cXt2rXm = as.formula(paste0(common, " +  ", country_time, " | OBJECTID  + as.factor(smllrgn):month | 0 | OBJECTID"))
+# formula (different intervention dummies for each temporal subsample)
+cXt2rXm = as.formula(paste0(common, " + I(intervention) +  ", country_time, " | OBJECTID  + as.factor(smllrgn):month | 0 | OBJECTID"))
+
+pre_data <- subset(complete, post1995==FALSE)
+pos_data <- subset(complete, post1995==TRUE)
 
-pre1995 = felm(data = subset(complete, post1995==FALSE), formula = cXt2rXm)
-post1995 = felm(data = subset(complete, post1995==TRUE), formula = cXt2rXm)
+pre1995 = felm(data = pre_data, formula = cXt2rXm)
```

Comment:
@cullen-molitor what's the idea behind doing it this way? seems less memory intensive to just subset within the ``felm`` call..